tapcfg/server.go

```diff
@@ -321,6 +324,19 @@ func genServerConfig(cfg *Config, cfgLogger btclog.Logger,
 
 	multiNotifier := proof.NewMultiArchiveNotifier(assetStore, multiverse)
 
+	// Construct the RFQ manager.
+	priceOracle := rfq.NewMockPriceOracle(3600)
```

Comment:
Should we add a TODO here? Or somehow else make sure a mock price oracle doesn't run in production?

Reply:
I've added a TODO.